provisioning/miniprov/hare_mp/cdf.py

```diff
@@ -37,11 +38,68 @@ def _create_node_descriptions(self) -> List[NodeDesc]:
             nodes.append(self._create_node(node))
         return nodes
 
+    def _create_pool_descriptions(self) -> List[PoolDesc]:
+        pools: List[PoolDesc] = []
+        conf = self.provider
+        cluster_id = conf.get('cluster>cluster_id')
+        storage_set_count = int(
+            conf.get(f'cluster>{cluster_id}>site>storage_set_count'))
+
+        for x in range(storage_set_count):
+            storage_set_name = conf.get(
+                f'cluster>{cluster_id}>storage_set{x+1}>name')
+
+            data_devices_count: int = 0
+            for node in conf.get(
+                    f'cluster>{cluster_id}>storage_set{x+1}>server_nodes'):
+                for device in conf.get(
+                        f'cluster>{node}>storage>data_devices'):
+                    data_devices_count += 1
+
+            data_units_count = int(conf.get(
+                f'cluster>{cluster_id}>storage_set{x+1}>durability>data'))
+            parity_units_count = int(conf.get(
+                f'cluster>{cluster_id}>storage_set{x+1}>durability>parity'))
+
+            #TODO We also need to add 'spare units' in below check
```

Comment:
we do have a key for spare units in conf-store `cluster>{cluster-id}>storage_set[N]>durability>spare` why not use it?
if not populated we can set it default to `parity_units`.

Reply:
Done